integrations/jina/src/haystack_integrations/components/embedders/jina/document_embedder.py

```diff
@@ -45,9 +47,9 @@ def __init__(
         embedding_separator: str = "\n",
     ):
         """
-        Create a JinaDocumentEmbedder component.
```

Comment:
Why remove this? Don't we have a line saying "Create a ... component." in most other components?